lib/blockchains/EthereumBlockchain.js

```diff
@@ -336,5 +369,95 @@ class EthereumBlockchain {
       return { valid: false, markAs: "failed", error: err.message }
     }
   }
+
+  async sendWithdrawalCoinsTransaction(coinAddress, totalAmount, hotWallet) {
+    const txToGetFee = await this.chain.new.Transaction()
+    const chainTransaction = await this.chain.new.Transaction()
+
+    try {
+      // To calculate fee let's add tx with the full amount
+      const txValues = { toAccountName: coinAddress, amount: totalAmount.toString(), symbol: 'ether' }
+      txToGetFee.actions = [await this.chain.composeAction(chainjs.Models.ChainActionType.ValueTransfer, txValues)]
+
+      const ethFee = await txToGetFee.getSuggestedFee(chainjs.Models.TxExecutionPriority.Fast)
+      const amountToTransfer = totalAmount.minus(new BigNumber(ethFee)).minus(new BigNumber(0.0001))
+      console.log(`Sending ${amountToTransfer.toString()} ETH to ${coinAddress}`)
+
+      // Set tx with updated amount
+      txValues.amount = amountToTransfer.toString()
+      chainTransaction.actions = [await this.chain.composeAction(chainjs.Models.ChainActionType.ValueTransfer, txValues)]
+
+      await chainTransaction.setDesiredFee(ethFee)
+      await chainTransaction.prepareToBeSigned()
+      await chainTransaction.validate()
+      await chainTransaction.sign([chainjs.HelpersEthereum.toEthereumPrivateKey(hotWallet.privateKey)])
+      const tx_result = await chainTransaction.send(chainjs.Models.ConfirmType.After001)
```

Comment:
I think we need at least 10 confirmations for ethereum, considering how critical this transaction is.